feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/rrm_config.c

```diff
@@ -274,8 +274,10 @@ void rrm_radio_rebalance_channel(const struct schema_Wifi_Radio_Config *rconf)
 
 	get_channel_bandwidth(mode, &channel_bandwidth);
```

Comment:
**get_channel_bandwidth(mode, &channel_bandwidth)** API return maximum channel bandwidth 80MHZ. It would be good add 160MHZ case as well to this API, since you are adding HE hardware mode, and 160 MHZ based support

Reply:
I think adding 160 Mhz should be dealt in a separate commit.